src/transform/convert/generate6XXFields.js

```diff
@@ -0,0 +1,19 @@
+export function handleTerms(marcRecord) {
```

Comment:
Why are we transferring subject headings from 'kaunokki' to 'ysa'? Why are we transferring genre/form headings from 'kaunokki' and 'bella' to 'slm/fin' and 'slm/fin'?
- YSA is not in use anymore
- kaunokki/bella and ysa or slm probably are NOT one to one matches?

Reply:
Kaunokki => sml/fin ok
Bella => sml/swe ok
Kaunokki => ysa => dropped ok